src/virtual_renderer.js

```diff
@@ -632,7 +631,7 @@ var VirtualRenderer = function(container, theme) {
         var posLeft = pixelPos.left;
         posTop -= config.offset;
 
-        var h = composition && composition.useTextareaForIME ? this.lineHeight : HIDE_TEXTAREA ? 0 : 1;
+        var h = composition && composition.useTextareaForIME || useragent.isMobile ? this.lineHeight : 1;
```

Comment:
`HIDE_TEXTAREA` was used to detect IE, we don't need IE specific modifications anymore?

Reply:
IE is not supported by microsoft, and we have removed most of ie specific code some time ago, this was just a leftover from that.